core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -827,14 +827,26 @@ Please use this method only if you are sure that the software is not working as
 If you have problems or questions, please review the FAQ on the \
 bisq.network web page or post in the Bisq forum in the Support section.
 
-support.initialInfo=Please note the basic rules for the dispute process:\n\
-1. You need to respond to the arbitrator's requests within 2 days.\n\
-2. The maximum period for a dispute is 14 days.\n\
-3. You need to cooperate with the arbitrator and provide the information they request to make your case.\n\
-4. You accepted the rules outlined in the wiki in the user agreement when you first started the application.\n\n\
-Please read more about the dispute process in our wiki:\nhttps://github.com/bisq-network/exchange/wiki/Arbitration-system
+support.initialInfo=Please send a description about your problem in the text field below. \
```

Comment:
Should be "Please **enter**..."

Reply:
Yes, I would make it "Please enter a description of your problem in the text field below."